internal/server/statvar/statvar_match.go

```diff
@@ -209,7 +209,7 @@ func BuildSQLiteIndex(
 	for index, doc := range docSet {
 		_, err = stmt.Exec(index, doc.ID, doc.Title, doc.Signature, doc.NumConstraints, doc.KeyValueText)
 		if err != nil {
-			fmt.Printf("Ignoring statvar with DCID=%s as its signature is not unique.\n", doc.ID)
+			// fmt.Printf("Ignoring statvar with DCID=%s as its signature is not unique.\n", doc.ID)
```

Comment:
@nopper I am seeing a lot of error logs from this. Is that expected?

Reply:
I am not sure. It looks like multiple statvars have the same set of of constraints and k/v properties. Is this expected?
The signature is computed here:

https://github.com/shifucun/mixer/blob/60da88e6693a1cf4253c25549567b32dad2efce7/internal/server/statvar/statvar_match.go#L169